doc/nrf/releases/release-notes-2.4.0.rst

```diff
@@ -0,0 +1,895 @@
+    If a configuration supports both Fast Pair and a dedicated dongle peer (:ref:`CONFIG_DESKTOP_BLE_DONGLE_PEER_ENABLE <config_desktop_app_options>`), the `Swift Pair`_ payload is, by default, included only for the dongle peer.
+  * Set the max compiled-in log level to ``warning`` for the Bluetooth HCI core (:kconfig:option:`CONFIG_BT_HCI_CORE_LOG_LEVEL`).
+    This is done to avoid flooding logs during application boot.
+  * The documentation with debug Fast Pair provisioning data obtained for development purposes.
+  * Aligned the nRF52833 dongle's board DTS configuration files and nRF Desktop's application-specific DTS overlays to hardware revision 0.2.1.
+
+Samples
+=======
+
+Bluetooth samples
+-----------------
+
+* :ref:`peripheral_hids_keyboard` and :ref:`peripheral_hids_mouse` samples:
+
+  * Updated the samples to register HID Service before Bluetooth is enabled (before calling the :c:func:`bt_enable` function).
+    The :c:func:`bt_gatt_service_register` function can no longer be called after enabling Bluetooth and before loading settings.
+
+* :ref:`peripheral_hids_mouse` sample:
+
+  * Updated the sample to include the :kconfig:option:`CONFIG_BT_SMP` Kconfig option when ``CONFIG_BT_HIDS_SECURITY_ENABLED`` is selected.
+  * Fixed a CMake warning by moving the nRF RPC configuration (the :kconfig:option:`CONFIG_NRF_RPC_THREAD_STACK_SIZE` Kconfig option) to a separate overlay config file.
+
+* :ref:`direct_test_mode` sample:
+
+  * Added:
+
+    * Support for the :ref:`nrfxlib:mpsl_fem` Tx power split feature.
+      The DTM command ``0x09`` for setting the transmitter power level takes into account the front-end module gain when this sample is built with support for front-end modules.
+      The vendor-specific commands for setting the SoC output power and the front-end module gain are not available when the :kconfig:option:`CONFIG_DTM_POWER_CONTROL_AUTOMATIC` Kconfig option is enabled.
+    * Support for +1 dBm, +2 dBm, and +3 dBm output power on the nRF5340 DK.
+
+  * Updated the handling of the hardware erratas.
+  * Removed a compilation warning when used with minimal pinout Skyworks FEM.
+
+* :ref:`peripheral_uart` sample:
+
+  * Fixed the unit of the :kconfig:option:`CONFIG_BT_NUS_UART_RX_WAIT_TIME` Kconfig option to comply with the UART API.
+
+* :ref:`nrf_dm` sample:
+
+  * Updated the sample by improving the scalability of the sample when it is being used with more devices.
+
+* :ref:`peripheral_fast_pair` sample:
+
+  * Added the default Fast Pair provisioning data that is used when no other provisioning data is specified.
+  * Updated the documentation to align it with the new way of displaying notifications for the Fast Pair debug Model IDs.
+
+* Removed the Bluetooth: External radio coexistence using 3-wire interface sample because of the removal of the 3-wire implementation.
+
+Bluetooth mesh samples
+----------------------
```

Comment:
PR https://github.com/nrfconnect/sdk-nrf/pull/11273 will affect this section

Reply:
Added the content under mesh samples section.